lib/node_modules/@stdlib/math/base/special/tribonacci/src/main.c

```diff
@@ -0,0 +1,103 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2022 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+#include "stdlib/math/base/special/tribonacci.h"
+#include "stdlib/constants/float64/max_safe_nth_tribonacci.h"
+
+static const int64_t tribonacci_value[ 64 ] = {
```

Comment:
```suggestion
static const int32_t STDLIB_CONSTANTS_FLOAT64_MAX_SAFE_NTH_TRIBONACCI = 63; // TODO: consider making a package similar to `@stdlib/constants/float64/max-safe-nth-fibonacci`
static const int64_t tribonacci_value[ 64 ] = {
```

Inlining the constant, as we need to actually create a package for it in order to work similar to Fibonacci.